chainer/exporters/caffe.py

```diff
@@ -383,7 +383,7 @@ def __call__(self, name, inputs, outputs):
                 for i in inputs[0].shape:
                     f.write(' dim: ' + str(i))
                 f.write(' } }\n'
-                        '} \n')
+                        '}\n')
```

Comment:
Note:
Good to fix the line, too, while it is a part of protobuf txt (the description in #8179 was not exact).